gha/bin/comment-error-to-pr.py

```diff
@@ -99,9 +98,15 @@ def handle(self) -> NoReturn:
                 expand_wrapper = f'<details>\n<summary>Expand {logfile_wc} lines long logs</summary>\n\n'
                 prepender = '\n'.join([prepender, expand_wrapper, '```'])
             prepend_in_file(logfile, prepender)                         # Forming comment body
-            with open(logfile, "a") as file:
+            with open(logfile, "a") as file:                            # Open file in append mode
                 if logfile_wc > 90:
-                    file.write("\n```\n</details>")
+                    error_entries = ''
+                    with open(logfile, "r") as file_read:               # Read all lines in a list in read mode
+                        lines = file_read.readlines()
+                        for line in lines:
+                            if line.find(ERROR_LOGS_KEY) != -1:
+                                error_entries = '\n'.join([error_entries, line])
+                    file.write(f"\n```\n</details>\n\n```\n{error_entries}```\n")
```

Comment:
```suggestion
                    error_entries = []
                    with open(logfile, "r") as file_read:               # Read all lines in a list in read mode
                        lines = file_read.readlines()
                        for line in lines:
                            if line.find(ERROR_LOGS_KEY) != -1:
                                error_entries.append(line)
                    file.write(f"\n```\n</details>\n\n```\n{"\n".join(error_entries)}```\n")
```